Cargo.toml

```diff
@@ -71,7 +71,7 @@ branch = "otak/fix-1.0.3"
 [patch.crates-io.proteus-traits]
 package = "proteus-traits"
 git = "https://github.com/wireapp/proteus"
-branch = "2.x"
+tag = "v2.1.1"
```

Comment:
Please get rid of the patches completely in our toplevel Cargo.toml.
The `otak/fix-1.0.3` thing should be pushed down to the only crate that actually uses it (and not as a patch). That crate does not use the workspace dependency anyway.

Reply:
The patch for the otak/fix cannot be removed, otherwise it conflicts with the one used by cryptobox in the interop crate. They're pointing to different commits...

Reply:
Why not update cryptobox to use the otak/fix-1.0.3 branch?